gtests/net/packetdrill/symbols_linux.c

```diff
@@ -190,6 +190,7 @@ struct int_symbol platform_symbols_table[] = {
 	{ TCP_TIMESTAMP,                    "TCP_TIMESTAMP"                   },
 	{ TCP_NOTSENT_LOWAT,                "TCP_NOTSENT_LOWAT"               },
 	{ TCP_INQ,			    "TCP_INQ"			      },
+	{ TCP_WINDOW_CLAMP,		    "TCP_WINDOW_CLAMP"		      },
```

Comment:
Thanks! Do you mind putting this after TCP_DEFER_ACCEPT and before TCP_INFO? That's where we have it in our internal sources. (And somehow that line didn't make it upstream... sorry!)